src/containers/AuthenticationPage/AuthenticationPage.js

```diff
@@ -217,12 +217,14 @@ export const AuthenticationForms = props => {
         />
       )}
 
-      {selectedRole === userTypes.normal && <SocialLoginButtonsMaybe
-        isLogin={isLogin}
-        showFacebookLogin={showFacebookLogin}
-        showGoogleLogin={showGoogleLogin}
-        from={from}
-      />}
+      {(selectedRole === userTypes.normal || isLogin) && (
```

Comment:
There are only 2 type of user, if it's normal user then it's acc, isn't this logic only required you to be login?

Reply:
There are 2 types of users: normal user and officer. The normal user can only book the service: rescue or adoption. The officer is the one who can create listings for the normal user to book: ACC. I just want the normal user can sign up with Google. An officer has to sign up by filling in the form, therefore, in the future, we can add more fields if we want the officer to provide more information during the signup stage.

Reply:
If that is the case, then checking selectedRole === userTypes.normal should already enough, we don't need isLogin

Reply:
Yes, I had already tried that, but because the SocialLoginButtonsMaybe component is the common component used for both LoginForm and SignupForm. Without the check for isLogin, when the normal users, who had already signed up with Google logged out and then they want to sign in again, they cannot find the SocialLoginButtonMaybe because, at that time, the selectedRole is null, because SignupForm and SignIn form are 2 different forms. So that I had to check for isLogin to ensure that the SocialLoginButtonMaybe will display when one of 2 conditions is met: a new user who ticked a radio button that has the value of normal user, or a user who wants to log in again.